packages/smooth_app/lib/database/dao_hive_product.dart

```diff
@@ -35,48 +34,4 @@ class DaoHiveProduct extends AbstractDao implements DaoProductMigrationSource {
 
   @override
   void registerAdapter() => Hive.registerAdapter(_ProductAdapter());
-
-  LazyBox<Product> _getBox() => Hive.lazyBox<Product>(_hiveBoxName);
-
-  Future<Product?> get(final String barcode) async => _getBox().get(barcode);
-
-  @override
-  Future<Map<String, Product>> getAll(final List<String> barcodes) async {
-    final LazyBox<Product> box = _getBox();
-    final Map<String, Product> result = <String, Product>{};
-    for (final String barcode in barcodes) {
-      final Product? product = await box.get(barcode);
-      if (product != null) {
-        result[barcode] = product;
-      }
-    }
-    return result;
-  }
-
-  Future<void> put(final Product product) async => putAll(<Product>[product]);
-
-  Future<void> putAll(final Iterable<Product> products) async {
-    final Map<String, Product> upserts = <String, Product>{};
-    for (final Product product in products) {
-      upserts[product.barcode!] = product;
-    }
-    await _getBox().putAll(upserts);
-  }
-
-  @override
-  Future<List<String>> getAllKeys() async {
-    final LazyBox<Product> box = _getBox();
-    final List<String> result = <String>[];
-    for (final dynamic key in box.keys) {
-      result.add(key.toString());
-    }
-    return result;
-  }
-
-  // Just for the migration
-  @override
-  Future<void> deleteAll(final List<String> barcodes) async {
-    final LazyBox<Product> box = _getBox();
-    await box.deleteAll(barcodes);
-  }
```

Comment:
Did you test if this part removal will work or also break the whole app, nontheless we should keep an eye out in the beta